### PR TITLE
tests(viewer): don't override puppeteer's chromium

### PR DESCRIPTION
**Pull request by connorjclark**
This test is currently failing in master.

The viewer's puppeteer test doesn't work with Chrome 78, and neither does the latest puppeteer (1.20.0).

Puppeteer ships with a specific Chromium revision, and is only meant to work with that, so we shouldn't override it.

### CODE REVIEW

**Comment by patrickhulce** [comment]
that didn't seem to do the trick yet

viewer tests failed with

>  Chromium revision is not downloaded. Run "npm install" or "yarn install"

**Comment by connorjclark** [comment]
MaybeTravis doesn't clear env variables between runs?

**Comment by brendankenny** [comment]
> Puppeteer ships with a specific Chromium revision, and is only meant to work with that, so we shouldn't override it.

this was partly to avoid redownloading Chromium every time (it may have been skipping `node_modules/` caching since pptr downloads it itself on install?) and making the travis runs even slower. Ideally someone in the world has figured that out since #4640?

Otherwise, is there something specific that broke with m78 and is it something we can get fixed upstream? pptr is going to have to update its version of Chromium at some point...

**Comment by connorjclark** [comment]
> Otherwise, is there something specific that broke with m78 and is it something we can get fixed upstream? pptr is going to have to update its version of Chromium at some point...

@mathiasbynens might know. [We could bisect it](https://docs.google.com/document/d/1GXpxna_EGRC8TkN6tLseuabfaxQLlWP2pQuQ73T-lUc/edit?ts=5daec46c).

**Comment by connorjclark** [comment]
```
tools/bisect-builds.py --use-local-cache -a linux64 -g 674921 -b 693954 --not-interactive -c 'bash -c "cd ~/code/lighthouse && CHROME_PATH=%p yarn jest viewer -t PSI"' --verify-range
```

Can't get the bad revision (`693954` is current stable) to fail, even though using the stable channel binary directly _does_ fail. Weird.

**Comment by connorjclark** [comment]
For travis, had to `unset` the puppeteer env variable. I guess it stuck around from previous runs.

File: .travis.yml [hunk 1]
```diff
@@ -17,10 +17,7 @@ cache:
 install:
   # Ensure we have the latest Chrome stable.
   - google-chrome-stable --version
-  # if our e2e tests fail in the future it might be that we are not compatible
-  # with the latest puppeteer api so we probably need to run on chromimum
-  # @see https://github.com/GoogleChrome/lighthouse/pull/4640/files#r171425004
-  - export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
+  - unset PUPPETEER_SKIP_CHROMIUM_DOWNLOAD
```
Comment (patrickhulce) [inline .travis.yml]: I'm assuming can we remove this at some point?

Comment (connorjclark) [reply .travis.yml]: yeah, but can't be sure at what point. unless someone wants to merge master into ever outstanding PR branch after this lands.

It sure is unexpected that environment variables seem to persist.

**Comment by connorjclark** [comment]
> a major bummer to our build times but a necessary evil to be able to keep merging I suppose

We could consider removing the other Chrome install, and just use Puppeteer's instead. We would have to manage updating Puppeteer on a regular basis. note: I don't love this idea.

**Comment by connorjclark** [comment]
One unexpected thing is that https://omahaproxy.appspot.com/ gives me `78.0.3904.70` as the stable version and `693954` as its base revision, but when the bisect script downloads `693954` it's actually version `78.0.3904.0`.

```
tools/bisect-builds.py --use-local-cache -a mac -g 674921 -b 693954 --not-interactive -c '%p --version' --verify-range
```
image

File: .travis.yml (hunk 1 above)
Comment (brendankenny) [reply .travis.yml]: we might need to just clear the caches. If we keep this we should also leave a comment similar to the removed one

Comment (connorjclark) [reply .travis.yml]: Clearing cache didn't help, I tried that first.

**Comment by connorjclark** [comment]
> does it work with the new 2.0.0 pptr release?

I don't think it will, since the issue is with Chromium and not Puppeteer. Let's see: https://github.com/GoogleChrome/lighthouse/pull/9886

File: .travis.yml (hunk 1 above)
Comment (brendankenny) [reply .travis.yml]: > environment variables seem to persist

I just don't see how this makes sense. It seems like people would be constantly running into it.

Comment (connorjclark) [reply .travis.yml]: Maybe b/c we should have use `env:` instead of a manual `export`?

**Pull request merged**